docs/index.rst

```diff
@@ -0,0 +1,42 @@
+.. alchemiscale-fah documentation master file, created by
+   sphinx-quickstart on Wed Feb 26 17:04:07 2025.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+#######################################################################
+high-throughput alchemical free energy execution...on Folding\@Home
+#######################################################################
+
+**alchemiscale-fah** provides deployable components for using `Folding@Home`_ work servers as compute sources for an `alchemiscale`_ server instance, enabling "planetary-scale" compute for :external+gufe:py:class:`~gufe.network.AlchemicalNetwork` evaluation.
+
+For users: see the :ref:`user-guide` for details on how to make use of Folding\@Home-based ``Protocol``\s to take advantage of 
```

Comment:
I think this sentence is incomplete - i.e. "take advantage of.. what?".

Reply:
Thank you! I ran off and worked on the user guide mid-sentence and never went back. :sweat_smile: